drake/systems/plants/KinematicsCache.h

```diff
@@ -121,6 +126,44 @@ class KinematicsCache
     }
   }
 
+  template <typename Derived>
+  Eigen::Matrix<typename Derived::Scalar, Derived::RowsAtCompileTime, Eigen::Dynamic> transformVelocityMappingToPositionDotMapping(const Eigen::MatrixBase<Derived>& mat) const
```

Comment:
Yeah, transforming Jacobians that were initially computed in terms of v to Jacobians that are in terms of qdot.

I'll add documentation.